server/lib/src/plugins/valuedeny.rs

```diff
@@ -19,9 +19,24 @@ impl Plugin for ValueDeny {
     ) -> Result<(), OperationError> {
         let denied_names = qs.denied_names();
 
+        if denied_names.is_empty() {
+            // Nothing to check.
+            return Ok(());
+        }
+
         let mut pass = true;
 
         for entry in cand {
+            // If the entry doesn't have a uuid, it's invalid anyway and will fail schema.
+            if let Some(e_uuid) = entry.get_uuid() {
+                // SAFETY - Thanks to JpWarren blowing his nipper clean off, we need to
+                // assert that the break glass accounts are NOT subject to this process.
+                if e_uuid == UUID_ADMIN || e_uuid == UUID_IDM_ADMIN {
+                    // These entries are exempt
+                    continue;
+                }
```

Comment:
I'd be nice if we also did a lookup on the account attributes to check in a more generic way whether they are system accounts or not. Happy to keep both as this guard is very cheap but It'd be nice to not run into this problem again when someone adds a third system account and forgets to update these lines